SunEngine.Core/Services/Sanitizer.cs

```diff
@@ -105,8 +101,18 @@ private void OnRemovingAttribute(object s, RemovingAttributeEventArgs e)
 
         private void OnRemovingTag(object sender, RemovingTagEventArgs e)
         {
+            var checkingTags = new[]
```

Comment:
It`s create many objects on heap. 
If this is a big problem, I can create a model and place an array declaration in the class.
Currently, there is no way to solve this problem in C# by saving the array declaration in method scope

Reply:
I think the best way is to make it static in class! static protectedor something like this